Assets/Resources/UI/Icon.uxml

```diff
@@ -0,0 +1,2 @@
+<ui:UXML xmlns:ui="UnityEngine.UIElements" editor-extension-mode="False" class="icon">
```

Comment:
Now much of a template is needed for an icon - do note that icon class that denotes that this is an icon component, which is a powerful way to be able to style it